### ISSUE TITLE
Cannot create volume from snapshot

### ISSUE THREAD

**Issue by igorgonibm** [kubernetes-csi/external-snapshotter#313]
We are adding snapshot support t our CSI driver for kubernetes 1.17
When trying to create volume from snapshot csi-provisioner can't fine volumesnapshot:
Describe on pvc gives:
`failed to provision volume with StorageClass "sc-snapshot": error getting handle for DataSource Type VolumeSnapshot by Name snap1: error getting snapshot snap1 from api server: the server could not find the requested resource (get volumesnapshots.snapshot.storage.k8s.io snap1)
`
But I can get the object:
```
kubectl get volumesnapshots.snapshot.storage.k8s.io snap1                         
NAME    READYTOUSE   SOURCEPVC      SOURCESNAPSHOTCONTENT   RESTORESIZE   SNAPSHOTCLASS   SNAPSHOTCONTENT                                    CREATIONTIME   AGE
snap1   true         pvc-snapshot                           1Gi           snapclass       snapcontent-07dfc3cc-120c-4a1e-9542-b3cecfb24b01   8d             8d

```
We use provisioner:    repository: quay.io/k8scsi/csi-provisioner    tag: "v1.4.0"

Attached yamls and logs
[vol_from_snap.zip](https://github.com/kubernetes-csi/external-snapshotter/files/4587544/vol_from_snap.zip)

**Comment by xing-yang** [comment]
External-provisioner v1.4.0 is released with k8s release 1.16 so it is too old.  You need to use at least v1.5.0.

**Comment by igorgonibm** [comment]
Thank you! With 1.5.0 it works.
